### ISSUE TITLE
Is it dead ?

### ISSUE THREAD

**Issue by dnxit** [ALMMa/datatables.mvc#96]
Hi, 

is this project dead ? it has a dependency on **Microsoft.AspNetCore.Mvc** which is deprecated and vulnerable as well, is there any plan on upgrading it ?

Kind Regards

**Comment by ALMMa** [comment]
I won't lie: today it is.
I prefer to take the honest path and tell you (all) this.

So, let me put into context.

I'm on a dead-end here now. I want to revamp this project but it would mean a full blown refactor from scratch. 
Not just an update.
I tried it several times, but every new time it took longer and longer, and maintaining compatibility to any level was getting to an impossible state.

When this project was created, it was meant to serve as a mapping for the old Hungarian Notation style. And right after that came the Camel Case with the version bump on DataTables, and I managed to make it work here as well.
However, last time I checked (and that's 2+ years ago) all details on the DataTables project, they had changed tons of things in there already.

I do appreciate that this gained attention, but I'm not sure if it's worth investing time anymore.

In between my current role which consumes a significant portion of my time and several personal and family issues that have been dragging for years now, I barely have time to look into this with the care it deserves. Even less for the rewrite it would require.

So, in simple terms, I'm taking the decision to archive this right now.
I cannot say this is a death of this project. But I want to make it explicit that it's not maintained anymore.

If time permits, once day, this might be brought back to life.
Or if someone shows clear intentions of catching up on this project I can provide proper access.

But until there, my suggestion would be to assume that yes, this is partially dead, stale and not actively maintained.
